src/emscripten-optimizer/parser.cpp

```diff
@@ -41,6 +41,7 @@ IString TOPLEVEL("toplevel"),
         CONTINUE("continue"),
         SWITCH("switch"),
         STRING("string"),
+        TRY("try"),
```

Comment:
I see, thanks. But not sure I fully understand. Do you emit normal JS outside of the asm.js module? Or inside? I didn't see test outputs here, which could have clarified this.